ci_framework/roles/build_containers/tasks/main.yml

```diff
@@ -13,6 +13,9 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
+- name: Include tasks from tasks.yml
+  ansible.builtin.include_tasks:
+    src: "{{ 'build.yml' if run_build|bool else 'hotfix.yml' }}"
```

Comment:
We can move this task just below line "Run container_build.sh script" task and make it conditional using `cifmw_build_containers_run_hotfix`.

Reply:
done